<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkFrame</class>
 <widget class="QFrame" name="NetworkFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="btOptions">
       <property name="toolTip">
        <string>Change Network Options</string>
       </property>
       <property name="statusTip">
        <string>Change Network Options</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ui.qrc">
         <normaloff>:/icons/images/preferences-system.svg</normaloff>:/icons/images/preferences-system.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btRuler">
       <property name="toolTip">
        <string>Change Network Scale</string>
       </property>
       <property name="statusTip">
        <string>Change Network Scale</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ui.qrc">
         <normaloff>:/icons/images/ruler.svg</normaloff>:/icons/images/ruler.svg</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btLock">
       <property name="toolTip">
        <string>Prevent the movement of nodes in this view</string>
       </property>
       <property name="statusTip">
        <string>Prevent the movement of nodes in this view</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ui.qrc">
         <normaloff>:/icons/images/lock.svg</normaloff>:/icons/images/lock.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="AnnotationsNetworkView" name="gvNetwork"/>
   </item>
   <item>
    <widget class="Slider" name="sliderScale">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Change scale of network graph</string>
     </property>
     <property name="statusTip">
      <string>Change scale of network graph</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>11</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnnotationsNetworkView</class>
   <extends>QGraphicsView</extends>
   <header>metgem_app/ui/widgets/annotations/view.h</header>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>metgem_app/ui/widgets/slider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ui.qrc"/>
 </resources>
 <connections/>
</ui>
